### PR TITLE
WIP: 🩹 Issue 74: fix deprecation warning

### PR DESCRIPTION
**Pull request by thomasmerz**
**Node.js 12 actions are deprecated**
For more information see:
https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.

> Please update the following actions to use Node.js 16: actions/checkout@v2
Using version v3, latest or master: actions/checkout#689

This PR fixes issue #74

### CODE REVIEW

**Comment by thomasmerz** [comment]
See before:
https://github.com/thomasmerz/productive-box/actions/runs/3688153259 ❌ 
And after:
https://github.com/thomasmerz/productive-box/actions/runs/3688200460 🟩 🚧

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ jobs:
       matrix:
         node_version: [14]
```
Comment (maxam2017) [inline .github/workflows/build.yml:13]:
```suggestion
        node_version: [16]
```

**Comment by maxam2017** [comment]
Those change has already been done by other PRs.
Thank you for your contribution and I hope you will consider submitting a new pull request in the future.

**Comment by thomasmerz** [comment]
@maxam2017 
> Those change has already been done by other PRs. Thank you for your contribution and I hope you will consider submitting a new pull request in the future.

Great, it's totally ok for me, that you used my issue/PR as an impuls to fix it yourself 👍🏼 

But you're still using and old version of `peter-evans/repository-dispatch@v1` which still uses Node 12. Please update this to [v2](https://github.com/peter-evans/repository-dispatch/releases/tag/v2.0.0), too.

**Comment by thomasmerz** [comment]
And why are all my ["Update gist" GH actions](https://github.com/thomasmerz/productive-box/actions/workflows/schedule.yml) failing since you merged your PR? 🤔

**Pull request closed without merging**